vmdb/app/models/container_service.rb

```diff
@@ -6,6 +6,7 @@ class ContainerService < ActiveRecord::Base
 
   belongs_to  :ext_management_system, :foreign_key => "ems_id"
   has_and_belongs_to_many :container_groups
+  has_many :container_service_port_configs
```

Comment:
@simon3z The problem with leaving it without a 'config' suffix is that its a little confusing. 
When I'm looking at a "service_port" field, what Im thinking is "This is a 4 digit port integer"
When I'm looking at "port_config" Im thinking about a data structure with multiple fields (which it is)